pkg/cmd/operator/analyze.go

```diff
@@ -104,9 +110,73 @@ func (o *AnalyzeOptions) Complete() error {
 	return nil
 }
 
+func (o *AnalyzeOptions) IsArchiveMode() bool {
+	return len(o.ArchivePath) != 0
+}
+
+func (o *AnalyzeOptions) IsKubeconfigMode() bool {
+	return len(o.Kubeconfig) != 0
+}
```

Comment:
This is incorrect - kubeconfig flag has a valid default (look into ClientConfig's Complete). So you are in "archive mode" if you provide archive path, otherwise you're in "kubeconfig mode". Also having two different functions for these is confusing - it's better to be clear that you can't be in both "modes" at once. I don't see much value in introducing the "mode", given you can just check for len(o.ArchivePath), but if that's your preference then I'd add a field in Options that you could switch on

Reply:
done

Reply:
still applies to an extent - I really don't think a wrapper for such a simple operation is justified. Imo either check against the flag directly, or add a type to Options that you could use `select` on